### PR TITLE
feat: bulk delete submission locks

### PR DESCRIPTION
**Pull request by nsprenkle**
**TL;DR -** add an endpoint for bulk deletion of submission locks. Used when exiting the new staff grading experience to reset in-progress locks.

JIRA: [AU-592](https://openedx.atlassian.net/browse/AU-592)

**What changed?**

- Add new XBlock JSON handler, `batch_delete_submission_lock` which takes a `submission_uuids` list of locks to try to delete.
- Endpoint will delete any locks in that list that the user owns, ignoring any locks owned by other users.
- Response is silent, returns empty `200` on success, `400` for bad request and `500` for internal errors

**Developer Checklist**

- [x] Reviewed the [release process](https://github.com/edx/edx-ora2/blob/master/.github/release_process.md)
- [x] Translations and JS/SASS compiled
- [ ] Bumped version number in [setup.py](https://github.com/edx/edx-ora2/blob/a62e81a9b0d89223476967ec3c27f3557a850735/setup.py#L39) and [package.json](https://github.com/edx/edx-ora2/blob/a62e81a9b0d89223476967ec3c27f3557a850735/package.json#L3)

**Testing Instructions**

- [ ] Unit tests
- [ ] Create submission locks under several users (can use Django or existing JSON handler endpoints)
- [ ] Exercise the endpoint with a list of `submission_uuids` including some (but not all) of your active locks, with other locks that are not owned by you.
- [ ] Verify that only the requested locks that you own are cleared

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [ ] I've done a visual code review
- [ ] I've tested the new functionality

FYI: @openedx/content-aurora

### CODE REVIEW

File: openassessment/staffgrader/models/submission_lock.py [hunk 1]
```diff
@@ -93,3 +93,14 @@ def clear_submission_lock(cls, submission_uuid, user_id):
             raise SubmissionLockContestedError
 
         current_lock.delete()
+
+    @classmethod
+    def batch_clear_submission_locks(cls, submission_uuids, user_id):
+        """
+        For a list of submission locks to try to clear, clear those that we own.
+
+        Returns: None or raises DB error
```
[the remark this answers is not in the document]
Comment (nsprenkle) [reply openassessment/staffgrader/models/submission_lock.py:102]: ^ Forgot I changed this during my fiddling. Will return the number deleted.

File: openassessment/staffgrader/staff_grader_mixin.py [hunk 2]
```diff
@@ -132,6 +132,31 @@ def delete_submission_lock(self, submission_uuid, data, suffix=''):  # pylint: d
...
+        """
+        Given a list of submission UUIDs, clear those that we currently have locks for.
+
+        Returns: None, no errors is implicit success
+
+        Raises:
+        - 400 in the case of bad params/data
+        - 500 for generic errors
+        """
+        anonymous_user_id = self.get_anonymous_user_id_from_xmodule_runtime()
```
[the remark this answers is not in the document]
Comment (nsprenkle) [reply openassessment/staffgrader/staff_grader_mixin.py:147]: Stupid question: is there a reason we don't want this class of error to be louder by default (i.e. no runtime or no anon ID = exceptions)? Feels like this is unnecessarily lenient and would be more likely to lead to errors like you're talking about.

For this case, happy to add that error checking but it seems like an annoying one-off since we aren't doing that in other places. I tend to like to follow the patterns established in the code so we have to keep fewer of these different cases in our heads.

**Review by jansenk — APPROVED** [review]
LGTM
